modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -181,6 +197,13 @@ public void testGetProcessInstancesPage() throws Exception {
 			instances -> assertEqualsIgnoringOrder(
 				Arrays.asList(instance1, instance2), instances));
 
+		_testGetProcessInstancesPage(
```

Comment:
@seltong, could you add the case when exists two sites but in just one the user will have permission to reviewer?